modules/persistent-storage-csi-snapshots-restore.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-snapshots.adoc
+
+[id="persistent-storage-csi-snapshots-restore_{context}"]
+= Restoring a volume snapshot
+
+Once you have a bound and ready VolumeSnapshot object, you can use that object to provision a new volume that is pre-populated with data from the snapshot.
+
+The volume snapshot content object is used to restore the existing volume to a previous state.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* An app; for example, `my-csi-app`.
```

Comment:
You don't need an app to do the restoration. This was used in the demo so I could indicate that the data has been retained.